lib/modules/apostrophe-templates/index.js

```diff
@@ -697,14 +697,15 @@ module.exports = {
         bodyClass;
     };
 
-    // Add a body attribute or attributes to be emitted when the page is rendered. This information
-    // is attached to `req.data`, where the string `req.data.aposBodyDataAttributes` is built up.
+    // Add a body attribute to be emitted when the page is rendered. This information
+    // is attached to `req.data`, where `req.data.aposBodyDataAttributes` is built up
+    // using `name` as the attribute name which is automatically prepended with "data-"
+    // and the optional `value` argument
     // The default `outerLayoutBase.html` template outputs that string.
-    // The string passed may contain space-separated attribute names which are automatically prepended with "data-".
 
-    self.addBodyDataAttribute = function(req, bodyAttribute) {
+    self.addBodyDataAttribute = function(req, name, value = '') {
       req.data.aposBodyDataAttributes = (req.data.aposBodyDataAttributes ? (req.data.aposBodyDataAttributes + ' ') : ' ') +
-        ('data-' + _.trim(bodyAttribute)).split(' ').join(' data-');
+        'data-' + (!_.isUndefined(value) ? (name + '=' + value + '') : name);
```

Comment:
I need to also check for `name` not being undefined / empty. Will push another commit for that, which will just return.

Reply:
Question goes. I wanted the value to be wrapped in `"` (double quotes) but it's being escaped in the body tag. How do I "bless" this internally to prevent that @boutell?